Week-2/Html Advanced/lineUp.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Select Line Up</title>
+</head>
+<body>
+    <form>
+        <p>
+            <fieldset>
+                <legend>Status:</legend>
+                <label >
+                    <input type="radio" name="status" value="Draft">
+                    Draft
+                </label>
+                <label>
+                    <input type="radio" name="status" value="Final" checked>
+                    Final
+                </label>
+            </fieldset>
+        </p>
+
+        <p>
+            <label for="gameDate">Date: </label>
+            <select name="gameDate" id="gameDate">
+                <option value="March 20">March 20</option>
+                <option value="March 24">March 24</option>
```

Comment:
Consider adding `aria-describedby`: You can enhance accessibility by adding the `aria-describedby` attribute to the select element, pointing to the id of a descriptive element that provides additional context or instructions about the date selection.